py/server/deephaven/table_factory.py

```diff
@@ -235,7 +234,6 @@ class InputTable(Table):
 
     Users should always create InputTables through factory methods rather than directly from the constructor.
     """
-    j_object_type = _JBaseArrayBackedInputTable
 
     def __init__(self, j_table: jpy.JType):
         super().__init__(j_table)
```

Comment:
I think we should validate that the `j_input_table` is of the correct Java class (`InputTableUpdater`), if possible.